storage/storage.go

```diff
@@ -36,12 +35,16 @@ var LicenseHandler Handler
 
 func Connect() {
 	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
-	MongoClient, err := mongo.Connect(ctx, options.Client().ApplyURI(config.Global.MongoURL))
+	MongoClient, err := mongo.Connect(ctx, options.Client().ApplyURI(config.Global.DBOptions.Mongo.URL))
 	fatalf("Problem while connecting to Mongo: %s", err)
 
-	LicenseHandler = licenseMongoHandler{MongoClient.Database(config.Global.DBName).Collection("licenses")}
-	GlobalKeyHandler = mongoKeyHandler{MongoClient.Database(config.Global.DBName).Collection("keys")}
-	//GlobalRSAHandler = mongoRSAHandler{MongoClient.Database(config.Global.DBName).Collection("keys")}
+	if config.Global.DBOptions.Type == "file" {
```

Comment:
I think this should be `mongo` and else should be `file` because we expect that if user doesn't set any storage type, it should fallback to `file` storage.